bower.json

```diff
@@ -1,10 +1,19 @@
 {
-  "name": "HabitRPG",
-  "version": "0.1.1",
```

Comment:
isn't the version number required? Also I'd just skip changes to bower.json, thanks but we're going to ditch it with the redesign that is in progress